README.md

```diff
@@ -706,6 +706,19 @@ read_by_id = client.read_note(written.note_id)
 print(read_by_id.data['lyrics'])
 ```
 
+## Identity
```

Comment:
I believe it also includes the identity story client

Reply:
Added a comment to make this explicit in the README